### PR TITLE
Add new files from #226.

### PR DESCRIPTION
**Pull request by obenland**
### Changes proposed in this Pull Request:

Adds template-related files and changes from #226.

### How to test the changes in this Pull Request:

Changes can't be tested since they're not called from anywhere.

### CODE REVIEW

File: newspack-blocks.php [hunk 1]
```diff
@@ -34,3 +34,33 @@ function newspack_blocks_plugin_textdomain() {
 	load_plugin_textdomain( 'newspack-blocks', false, dirname( plugin_basename( __FILE__ ) ) . '/languages' );
 }
 add_action( 'plugins_loaded', 'newspack_blocks_plugin_textdomain' );
+
```
Comment (marekhrabe) [inline newspack-blocks.php]: Please move function definitions out of `newspack-blocks.php`. We have done that to all other functions recently as part of refactoring and they all went into `class-newspack-blocks.php`

File: src/blocks/homepage-articles/articles-list-amp.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
```
Comment (marekhrabe) [inline src/blocks/homepage-articles/articles-list-amp.php]: Did we end up using AMP? If not, can we delete this file?

Comment (WunderBart) [reply src/blocks/homepage-articles/articles-list-amp.php]: We've ended up not supporting the `Load more` functionality for AMP requests, which means we're not using any AMP components at this point. I think this one can be safely deleted.

cc @getdave

File: src/blocks/homepage-articles/article.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+	$styles = 'min-height: ' . $attributes['minHeight'] . 'vh; padding-top: ' . ( $attributes['minHeight'] / 5 ) . 'vh;';
+}
+
+$image_size = 'newspack-article-block-uncropped';
+if ( 'uncropped' !== $attributes['imageShape'] ) {
+	$image_size = Newspack_Blocks::image_size_for_orientation( $attributes['imageShape'] );
+}
+
+$thumbnail_args = '';
+// If the image position is behind, pass the object-fit setting to maintain styles with AMP.
```
Comment (WunderBart) [inline src/blocks/homepage-articles/article.php]: Is this comment still relevant?

**Comment by jeffersonrabb** [comment]
Superseded by https://github.com/Automattic/newspack-blocks/pull/255, https://github.com/Automattic/newspack-blocks/pull/250, https://github.com/Automattic/newspack-blocks/pull/248, https://github.com/Automattic/newspack-blocks/pull/259. Closing.

**Pull request closed without merging**